layer.yaml

```diff
@@ -12,6 +12,7 @@ includes:
   - 'layer:vault-kv'
   - 'layer:vaultlocker'
   - 'layer:hacluster'
+  - 'interface:cluster-dns'
```

Comment:
Make sure you add this new interface to the [layer index](https://github.com/juju/layer-index)

Reply:
[Done!](https://github.com/juju/layer-index/pull/101)